Task_DEV-13/src/main/java/Criteria/Cost.java

```diff
@@ -0,0 +1,65 @@
+package Criteria;
+
+import Employees.Group;
+
+import java.util.*;
+
+/**
+ * Criterion2: The minimum cost for a fixed productivity.
+ */
+public class Cost {
```

Comment:
I expect that all criteria should implement some common interface or at lest extend some common class